### ISSUE TITLE
Support for OIDC secrets

### ISSUE THREAD

**Issue by nbrinks** [codesquadnest/zuulcilint#267]
Currently, `oidc` secrets are flagged as invalid.

## Example

`zuul.d/secret.yaml`
```yaml
- secret:
    name: my_oidc_token
    oidc:
```

## Expected

`oidc` secrets validated against schema defined by https://zuul-ci.org/docs/zuul/latest/config/secret.html#attr-secret.oidc

## Actual

`oidc` secrets are not supported

```
Message: Additional properties are not allowed ('oidc' was unexpected)
```

**Comment by martimmatias** [comment]
Hi, I added this to my todo list.

This is likely caused by the schema being out of date...
